packages/cf.js/test/provider.spec.ts

```diff
@@ -22,6 +30,16 @@ class TestNodeProvider implements INodeProvider {
   }
 }
 
+const TEST_APP_INSTANCE_INFO: AppInstanceInfo = {
```

Comment:
This seems like a generic dummy that could be used for other tests. Might be worth putting it in a `common.ts` or somewhere.